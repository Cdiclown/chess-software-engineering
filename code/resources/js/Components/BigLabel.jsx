export default function (props)
{
    return (
        <h2 className='label label-big'>{props.children}</h2>
    );
}
