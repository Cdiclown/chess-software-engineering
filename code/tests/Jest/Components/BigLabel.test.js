import BigLabel from "../../../resources/js/Components/BigLabel.jsx";
import renderer from 'react-test-renderer';

it("text renders correctly", () => {

    const component = renderer.create(
        <BigLabel>Test2</BigLabel>
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
