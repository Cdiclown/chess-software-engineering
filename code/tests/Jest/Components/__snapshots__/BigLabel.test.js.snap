// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`text renders correctly 1`] = `
<h2
  className="label label-big"
>
  Test2
</h2>
`;
